openpype/modules/clockify/launcher_actions/ClockifyStart.py

```diff
@@ -6,9 +6,12 @@
 class ClockifyStart(LauncherAction):
     name = "clockify_start_timer"
     label = "Clockify - Start Timer"
-    icon = "clockify_icon"
+    icon = "app_icons/clockify.png"
     order = 500
     clockapi = ClockifyAPI()
+    clockapi.set_api()
```

Comment:
This will trigger logic on import which is not good. The api should be initialized only when the action is used. Also the workspace of user may change between import and triggering the process. Also if the api initiaization would crash it would crash the action import and user can't know why he does not see the action.

Reply:
This is for all the actions.